services-camel-app/src/test/java/edu/unc/lib/boxc/services/camel/images/AddDerivativeProcessorTest.java

```diff
@@ -1,53 +1,50 @@
 package edu.unc.lib.boxc.services.camel.images;
 
+import edu.unc.lib.boxc.model.fcrepo.ids.PIDs;
+import edu.unc.lib.boxc.model.fcrepo.test.TestHelper;
+import edu.unc.lib.boxc.services.camel.images.AddDerivativeProcessor.DerivativeGenerationException;
+import edu.unc.lib.boxc.services.camel.util.CdrFcrepoHeaders;
+import org.apache.camel.Exchange;
+import org.apache.camel.Message;
+import org.apache.camel.component.exec.ExecResult;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.io.TempDir;
+import org.mockito.Mock;
+
+import java.io.ByteArrayInputStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Arrays;
+
 import static edu.unc.lib.boxc.services.camel.util.CdrFcrepoHeaders.CdrBinaryMimeType;
 import static org.fcrepo.camel.FcrepoHeaders.FCREPO_URI;
 import static org.junit.Assert.assertFalse;
```

Comment:
Junit 5 imports are `org.junit.jupiter.api.Assertions.assertFalse`, `org.junit.jupiter.api.Assertions.assertThrows`, and `org.junit.jupiter.api.Assertions.assertTrue`. I'm not sure if we need to change this since junit vintage is active.